snippets/README.md

```diff
@@ -51,7 +51,7 @@ naming conventions:
 - File extensions should show which language they are written in (except for
   english which is the default)
 - Files cannot be translated until there is an english version, so that everyone
-  can share this study guide
```

Comment:
I think this change happend by accident.

Reply:
Yes ;)